lib/plug/adapters/cowboy.ex

```diff
@@ -117,6 +117,19 @@ defmodule Plug.Adapters.Cowboy do
     :ranch.child_spec(ref, nb_acceptors, ranch_module, trans_opts, :cowboy_protocol, proto_opts)
   end
 
+  @doc """
+  Returns runtime info about the Ranch listener given by `ref`.
+
+  For more information, see `ranch:get_port/1`, `ranch:get_max_connections/1`, 
+  and `ranch:get_protocol_options/1` in the [Ranch 1.1 
+  documentation](http://ninenines.eu/docs/en/ranch/1.1/manual/ranch/index.html).
+  """
```

Comment:
should we `spec` this or at least give a possible keyspace of `info` that other adapters are expected to provide? It won't be the same for all adopters, but some subset of keys should be implemented for each adapter.